### PR TITLE
More clarity on dispensing state

### PR DESCRIPTION
**Pull request by guilhermedemouraa**

### CODE REVIEW

File: py/farm_ng/canbus/packet.py [hunk 1]
```diff
@@ -598,6 +598,6 @@ def from_raw_canbus_message(cls, message: canbus_pb2.RawCanbusMessage) -> BugDis
     def __str__(self):
         """Returns a string representation of the class."""
         return (
-            f"BugDispenserTpdo1: Rates: {self.rate0}, {self.rate1}, {self.rate2} "
+            f"BugDispenserTpdo3: Rates: {self.rate0}, {self.rate1}, {self.rate2} "
```
Comment (Hackerman342) [inline py/farm_ng/canbus/packet.py:601]: This message is not a "tpdo3", it is a `BugDispenserState` message that happens to be sent on a `Tpdo3` channel

```suggestion
            f"BugDispenserState: Rates: {self.rate0}, {self.rate1}, {self.rate2} "
```

File: py/farm_ng/canbus/packet.py [hunk 2]
```diff
@@ -528,7 +528,7 @@ def decode(self, data):
 
     def __str__(self):
         """Returns a string representation of the class."""
-        return f"BugDispenserRpdo1: Rates: {self.rate0}, {self.rate1}, {self.rate2}"
+        return f"BugDispenserRpdo3: Rates: {self.rate0}, {self.rate1}, {self.rate2}"
```
Comment (Hackerman342) [inline py/farm_ng/canbus/packet.py:531]: ditto

File: protos/farm_ng/canbus/tool_control.proto [hunk 3]
```diff
@@ -38,8 +38,14 @@ enum PtoCommandType {
...
-  BUG_DISPENSER_STOPPED = 2; // Stop the bug dispenser
+  BUG_DISPENSER_ACTIVE = 1; // The bug dispenser is active
+  BUG_DISPENSER_STOPPED = 2; // The bug dispenser is stopped
+}
+
+enum BugDispenserIsDispensing {
+  DISPENSING_UNKNOWN = 0;
+  DISPENSING_TRUE = 1;
+  DISPENSING_FALSE = 2;
 }
```
Comment (Hackerman342) [inline protos/farm_ng/canbus/tool_control.proto:49]: This seems a little over-complicated? Would we actually change behavior for unknown vs defaulting to false? If it is a need to have this level of granularity, then ignore me

**Review by Hackerman342 — APPROVED** [review]
Some comments, but feel free to merge after

File: protos/farm_ng/canbus/tool_control.proto (hunk 3 above)
Comment (guilhermedemouraa) [reply protos/farm_ng/canbus/tool_control.proto]: Yeah, I think we can just default the state to false if we're not confident we're dispensing. Suggestion implemented on commit 5e58f13e3aeafe3f8d84d76eebc7b8670e97b799

**Pull request merged**